strongbox-web-core/src/test/java/org/carlspring/strongbox/security/authentication/StrongboxAuthenticationFilterTest.java

```diff
@@ -0,0 +1,82 @@
+package org.carlspring.strongbox.security.authentication;
+
+import org.carlspring.strongbox.security.authentication.strategy.DelegatingAuthenticationConverter;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.core.Authentication;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+class StrongboxAuthenticationFilterTest
```

Comment:
why not `public` class?

Reply:
habit from using junit jupiter in other projects. I can add public

Reply:
I think it should be public